Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/Scripts/Exporters/IUnityProjectExporter.cs

```diff
@@ -40,10 +40,10 @@ public interface IProjectExporter
         void ExportSolution(UnityProjectInfo unityProjectInfo, MSBuildToolsConfig config);
 
         /// <summary>
-        /// Generates the Directory.Build.props file that is expected to be used by both generated and non-generated projects alike.
+        /// Creates an exporter for the commom MSBuild file that is expected to be used by both generated and non-generated projects alike.
```

Comment:
>commom MSBuild [](start = 40, length = 14)

nit: it might make more sense to actually call out the file name here if its the same across projects